src/muxy.ts

```diff
@@ -30,6 +31,7 @@ export interface SDKMap {
  */
 export interface SetupOptions {
   clientID: string;
+  purchaseClientEnabled: boolean;
```

Comment:
Don't call this `purchaseClientEnabled`, that is an internal name not one the user needs to be concerned with. Please use `transactionsEnabled` as before.